tests/performance/apdex-score/wdio.conf.js

```diff
@@ -0,0 +1,335 @@
+require('../../aliases');
+const loadSettings = require('./settings-provider');
+
+exports.config = {
+  //
+  // ====================
+  // Runner Configuration
+  // ====================
+  // WebdriverIO supports running e2e tests as well as unit and component tests.
+  runner: 'local',
+  port: 4723,
+  //
+
+  services: [
+    ['appium', { command: 'appium', logPath: './logs/' }]
```

Comment:
This creates a new folder in the root level of CHT, it can be confusing. 
We are saving the automation logs inside /tests/ folder, for example see [here](https://github.com/medic/cht-core/blob/5274b92dc25d54680427af8ad55ae7024485113e/tests/wdio.conf.js#L14).

Reply:
sure